packages/discovery-react-components/package.json

```diff
@@ -61,6 +61,7 @@
     "@ibm-watson/discovery-styles": "1.x.x",
     "carbon-components": ">= 10.6.0 < 11",
     "carbon-components-react": ">= 7.6.0 < 8",
+    "core-js": "^2.6.12",
```

Comment:
Example app was failing to properly load the `discovery-react-components` library, saying it couldn't find `core-js/es6/symbol`. This is due to `react-virtualized` still using `core-js` v2.x, which is no longer supported. I had to add `core-js` as a peer-dep here and as a dep for the example app.

Unfortunately, `react-virtualized` is no longer maintained and there are other of its dependencies which will cause us issues. We will have to find a suitable replacement, soon.